policy/policy.sample.json

```diff
@@ -0,0 +1,78 @@
+{
+    "SecurityPolicies": [
+        {
+            "Name": "policy1",
+            "PolicyID": "3710940D-FBC0-4A44-A95D-E549E628B4DF", # unique identifier across tenants, auto generated for POST.
+            "AppliedTo": [ # can attach multiple tenants to which the policy can be applied to.
```

Comment:
Do we need "appliedTo" though or tenants/segments can just be speciifers of sort and destination?